homeassistant/components/todoist/calendar.py

```diff
@@ -541,9 +541,8 @@ def create_todoist_task(self, data: Task):
             return None
 
         # All task Labels (optional parameter).
-        task[LABELS] = [
-            label.name for label in self._labels if label.name in data.labels
-        ]
+        labels = data.labels or []
+        task[LABELS] = [label.name for label in self._labels if label.name in labels]
```

Comment:
Just fixing a mypy error: `Error: homeassistant/components/todoist/calendar.py:545: error: Unsupported right operand type for in ("list[str] | None")  [operator]` since `data.labels` could be `list[str] | None`.